src/sql.rs

```diff
@@ -1649,7 +1647,9 @@ mod test {
         assert_eq!(avatar_bytes, &async_std::fs::read(&a).await.unwrap()[..]);
 
         t.sql.close().await;
-        housekeeping(&t).await;
+
+        // Don't check return type, this is not supposed to successfully run, we just want to make sure that no files are deleted
```

Comment:
Maybe just `if let Err(err)` and `warn`/`info`? This comment will be hard to understand for someone reading it several months later who don't know that housekeeping was once unresultified, ignored inner errors and corrupted the database.

Reply:
Actually it's supposed to return an Err, so now I'm calling `unwrap_err()`